### PR TITLE
Update for 'single-threaded' property

### PR DESCRIPTION
**Pull request by patilatharva**
In accordance with the PR to replace `threading` with `single-threaded`: https://github.com/lf-lang/lingua-franca/pull/1817.

### CODE REVIEW

File: packages/documentation/copy/en/reference/Target Declaration.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ A target specification may have optional parameters, the names and values of whi
 - [**runtime-version**](#runtime-version): Specify which version of the runtime system to use.
 - [**rust-include**](#rust-include): (Rust only) A set of Rust modules in the generated project.
 - [**single-file-project**](#single-file-project): (Rust only) If true, enables [single-file project layout](#single-file-layout).
-- [**threading**](#threading): Whether to use multiple threads.
+- [**single-threaded**](#single-threaded): Whether to use a single-threaded or multi-threaded runtime.
```
Comment (lhstrh) [inline packages/documentation/copy/en/reference/Target Declaration.md:44]:
```suggestion
- [**single-threaded**](#single-threaded): Whether to use a single-threaded runtime (as opposed to a multi-threaded one).
```

**Review by lhstrh — APPROVED** [review]
Looks great! Thanks for this! I only made one suggestion to resolve same ambiguity. Let's wait with merging until the changes are in effect (and released).

File: packages/documentation/copy/en/reference/Target Declaration.md (hunk 1 above)
Comment (patilatharva) [reply packages/documentation/copy/en/reference/Target Declaration.md:44]: Thanks for the edit! I've made the other PR ready for review.

**Comment by lhstrh** [comment]
Note to @patilatharva, I copied the text from #145 and force pushed it here because it is an edited version of the changes that you proposed here earlier. Because we didn't manage to merge in your PRs before the 0.5.0 release, we cannot include the text in #145.

**Comment by lhstrh** [comment]
@axmmisaka: the associated PR in `lingua-franca` is not ready to merge, and these changes to the documentation need to be staged for the next release. Have you been able to prototype a mechanism for this?

**Comment by axmmisaka** [comment]
> @axmmisaka: the associated PR in `lingua-franca` is not ready to merge, and these changes to the documentation need to be staged for the next release. Have you been able to prototype a mechanism for this?

Apologies for the delay! I have a PoC here: https://github.com/axmmisaka/lf-lang.github.io/tree/versioning, and I expect to bring it to a good working state today.

**Comment by lhstrh** [comment]
Subsumed by #224.

**Pull request closed without merging**